tests/parsing/errors/other/__snapshots__/parse.spec.js.snap

```diff
@@ -216,14 +219,36 @@ Solution: you need to pull out each field you want explicitly.
 
   Syntax error!
   parsing/errors/other/spread.js:8:13-22  
-  6 │ 
-  7 │ let myList = list{...x, ...y}
-  8 │ let list{...x, ...y} = myList
-  9 │ 
+   6 │ 
+   7 │ let myList = list{...x, ...y}
+   8 │ let list{...x, ...y} = myList
+   9 │ 
+  10 │ type t = {...a}
   
   List pattern matches only supports one \`...\` spread, at the end.
 Explanation: a list spread at the tail is efficient, but a spread in the middle would create new list[s]; out of performance concern, our pattern matching currently guarantees to never create new intermediate data.
 
 
+  Syntax error!
+  parsing/errors/other/spread.js:10:11-13  
+   8 │ let list{...x, ...y} = myList
+   9 │ 
+  10 │ type t = {...a}
```

Comment:
Could the error be

```
You are using the spread `...` operator without extending the type, you can alias the type if you want it to be the same. 
```

or am I missing some case there?

Reply:
Thanks for the suggestion! Will implement.

Reply:
The general difficulty here is that we don't know whether it's an object or record…

Reply:
Yeah, in case the user makes the error on records, it'd give a two-step error:

```rescript
type b = {...a}
```

> **Error**: You are using the spread `...` operator without extending the type, you can alias the type if you want it to be the same.

```rescript
type b = {...a, x: string}
```

> **Error**: A record type declaration doesn't support the ... spread. Only an object (with quoted field names) does.

For sure it's a bit more cumbersome but I feel like this "teaches" the context where `...` applies better.